src/expand.cpp

```diff
@@ -104,7 +104,7 @@ namespace Sass {
     bool has_parent_selector = false;
     for (size_t i = 0, L = selector_stack.size(); i < L && !has_parent_selector; i++) {
       Selector_List_Obj ll = selector_stack.at(i);
-      has_parent_selector = ll != 0 && ll->length() > 0;
+      has_parent_selector = ll != nullptr && ll->length() > 0;
```

Comment:
All of these comparisons with `0` were bugs (never worked as intended)!

Reply:
I guess in other places we have used `isNull` of `bool` operator ...